<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
    <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Open+Sans">
    <LINK REL=StyleSheet HREF="main.css" TYPE="text/css">
    <title>Research of Vladimir Minin</title>
  </head>

  <body>

    <div class="row">
      <div class="col-1"></div>
      <div class="col-3">
        <nav>
            <ul id="homebar">
              <li>Vladimir N. Minin</li>
            </ul>
          </nav>
        </div>

      <div class="col-8">

        <nav>
            <ul id="navbar">
              <li><a href="index.html">Main</a></li>
              <li class="active"><a href="research.html">Research</a></li>
              <li><a href="software.html">Software</a></li>
              <li><a href="mentoring.html">Mentoring</a></li>
              <li><a href="teaching.html">Teaching</a></li>
              <li><a href="https://github.com/vnminin/stoch_mod_journal_club" target="_blank">Group Meetings</a></li>
            </ul>
          </nav>
      </div>


  </div>

    <div class="row">

      <div class="col-1">
      </div>

      <div class="col-9">

  <h3>Preprints (highlighted are <font color=#8A2BE2>student</font> collaborators):</h3>

  <ol id="pubs" reversed>
    <li><b><font color=#8A2BE2>Xu J</font></b>, Koelle S, Guttorp G, Wu C, Dunbar CE,  Abkowitz JL, <b>Minin VN</b>.
      Statistical inference in partially observed stochastic compartmental models with application to cell lineage tracking of in vivo hematopoiesis,
      <a href="https://arxiv.org/abs/1610.07550" target="_blank">arxiv:1610.07550</a>,
      <a href="papers/Xu2016.pdf" target="_blank"><b>PDF</b></a>,
      software: <a href="https://github.com/jasonxu90/branchCorr" target="_blank">branchCorr</a>.

  </ol>

<h3>Book Chapters (highlighted are <font color=#8A2BE2>student</font> collaborators):</h3>

<ol id="pubs" reversed>
  <li><b><font color=#8A2BE2>Dhar A</font></b> and <b>Minin VN</b>.
    Maximum likelihood methods for phylogenetic inference,
    <a href="http://www.sciencedirect.com/science/article/pii/B9780128000496002079" target="_blank">Encyclopedia of Evolutionary Biology</a>, edited by Kliman P (Phylogenetic Methods section edited by Kubatko L), 499 &#x2013 506,
    2016, <a href="papers/Dhar2016.pdf" target="_blank"><b>PDF</b></a>.
  <li><b><font color=#8A2BE2>Palacios JA</font></b>, Gill MS, Suchard MA, and <b>Minin VN</b>.
    Bayesian nonparametric phylodynamics, in
    <a href="http://www.crcpress.com/product/isbn/9781466500792" target="_blank">Bayesian Phylogenetics: Methods, Algorithms, and Applications</a>,
    edited by Chen MH, Kuo L, Lewis PO, 2014.
</ol>

<h3>Refereed Articles (highlighted are <font color=#8A2BE2>student</font> and <font color=#917B4C>postdoc</font>
  collaborators):</h3>

<ol id="pubs" reversed>
  <li class="year">2017 and in press

    <li class="afteryear">Ho LST, <b><font color=#8A2BE2>Xu J</font></b>, Crawford FW, <b>Minin VN</b>, Suchard, MA.
      Birth(death)/birth-death processes and their computable transition probabilities with statistical applications,
      <i><a href="https://link.springer.com/article/10.1007/s00285-017-1160-3" target="_blank">Journal of Mathematical Biology</a></i>,
      in press,
      <a href="http://arxiv.org/abs/1603.03819" target="_blank">arXiv:1603.03819</a>,
      <a href="papers/Ho2017.pdf" target="_blank"><b>PDF</b></a>,
      software: <a href="https://github.com/msuchard/MultiBD" target="_blank">MultiBD</a>.

    <li> Hardin WR, Li R, Xu J, Shelton AM, Alas GCM, <b>Minin VN</b>, Paredez AR.
    Myosin-independent cytokinesis in <i>Giardia</i> utilizes flagella to coordinate force generation and direct membrane trafficking,
    <i><a href="http://www.pnas.org/content/early/2017/07/03/1705096114.abstract" target="_blank">Proceedings of the National Academy of Sciences, USA</a></i>, in press.

    <li> <b><font color=#8A2BE2>Fintzi J</font></b>, <b><font color=#8A2BE2>Cui X</font></b>, Wakefield J, <b>Minin VN</b>.
      Efficient data augmentation for fitting stochastic epidemic models to prevalence data,
      <i><a href="http://dx.doi.org/10.1080/10618600.2017.1328365" target="_blank">Journal of Computational and Graphical Statistics</a></i>, in press,
      <a href="http://arxiv.org/abs/1606.07995" target="_blank">arxiv:1606.07995</a>,
      <a href="papers/Fintzi2016.pdf" target="_blank"><b>PDF</b></a>,
      software: <a href="https://github.com/fintzij/BDAepimodel" target="_blank">BDAepimodel</a>.
      <font color="brown"><b>With a shorter version of this paper, Jon Fintzi
        won 2016 WNAR student written paper competition.</b></font>

    <li><b><font color=#8A2BE2>Faulkner JR</font></b> and <b>Minin VN</b>.
      Locally adaptive smoothing with Markov random fields and shrinkage priors,
      <i><a href="http://projecteuclid.org/euclid.ba/1487905413" target="_blank">Bayesian Analysis</a></i>, in press,
      <a href="http://arxiv.org/abs/1512.06505"target="_blank">arXiv:1512.06505</a>,
      <a href="papers/Faulkner2015.pdf" target="_blank"><b>PDF</b></a>,
      software: <a href="https://github.com/jrfaulkner/bnps" target="_blank">bnps</a>.
      <font color="brown"><b>With a shorter version of this paper, Jim Faulkner was a runner-up (Distinguished Student Paper Award) in 2015
      WNAR student written paper competition.</b></font>

    <li><b><font color=#8A2BE2>Dhar A</font></b> and <b>Minin VN</b>.
      Calculating higher-order moments of phylogenetic stochastic mapping summaries in linear time,
      <i><a href="http://online.liebertpub.com/doi/10.1089/cmb.2016.0172" target="_blank">Journal of Computational Biology</a></i>, 24, 377 &#x2013 399,
      <a href="http://arxiv.org/abs/1609.07844" target="_blank">arxiv:1609.07844</a>,
      <a href="papers/Dhar2016b.pdf" target="_blank"><b>PDF</b></a>,
      software: <a href="https://github.com/dunleavy005/phylomoments" target="_blank">phylomoments</a>.

    <li><b><font color=#8A2BE2>Karcher M</font></b>, Palacios JA, Lan S, <b>Minin VN</b>.
     phylodyn: an R package for phylodynamic simulation and inference,
     <i><a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12630/full" target="_blank">Molecular Ecology Resources</a></i>,
     17, 96 &#x2013 100,
     <a href="https://arxiv.org/abs/1610.05817"target="_blank">arXiv:1610.05817</a>,
     <a href="papers/Karcher2016b.pdf" target="_blank"><b>PDF</b></a>,
     software: <a href="https://github.com/mdkarcher/phylodyn" target="_blank">phylodyn</a>.

     <li class="year">2016

    <li class="afteryear"><b><font color=#8A2BE2>Koepke AA</font></b>, Longini, IM, Halloran ME, Wakefield J, <b>Minin VN</b>.
    Predictive modeling of cholera outbreaks in Bangladesh,
    <i><a href="http://imstat.org/aoas/next_issue.html" target="_blank">Annals of Applied Statistics</a></i>, 10,  575 &#x2013 595,
    <a href="http://arxiv.org/abs/1402.0536"target="_blank">arXiv:1402.0536</a>,
    <a href="papers/Koepke2016.pdf" target="_blank"><b>PDF</b></a>,
    software: <a href="https://github.com/vnminin/bayessir" target="_blank">bayessir</a>.
    <font color="brown"><b>With a shorter version of this paper, Amanda Koepke won a 2015 Young Investigator
    Award from ASA Statistics in Epidemiology Section.</b></font>

  <li><b><font color=#917B4C>Linkem CW</font></b>, <b>Minin VN</b>, Leach&eacute AD.
    Detecting the anomaly zone in species trees and evidence for a misleading signal in higher-level skink phylogeny (Squamata: Scincidae),
    <i><a href="http://sysbio.oxfordjournals.org/content/65/3/465" target="_blank">Systematic Biology</a></i>, 65, 465 &#x2013 477,
    <a href="http://biorxiv.org/content/early/2014/12/03/012096" target="_blank">bioRxiv</a>,
    <a href="papers/Linkem2016.pdf" target="_blank"><b>PDF</b></a>.

    <li><b><font color=#8A2BE2>Karcher M</font></b>, Palacios JA, Bedford T, Suchard MA, <b>Minin VN</b>.
     Quantifying and mitigating the effect of preferential sampling on phylodynamic inference,
     <i><a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004789" target="_blank">PLOS Computational Biology</a></i>, 12, e1004789,
     <a href="http://arxiv.org/abs/1510.00775"target="_blank">arXiv:1510.00775</a>,
     <a href="papers/Karcher2016.pdf" target="_blank"><b>PDF</b></a>,
     software: <a href="https://github.com/mdkarcher/phylodyn" target="_blank">phylodyn</a>.

  <li class="year">2015

  <li class="afteryear"><b><font color=#8A2BE2>Xu J</font></b>, Guttorp P, Kato-Maede M, <b>Minin VN</b>.
	Likelihood-based inference for discretely observed birth-death-shift processes, with applications to evolution of mobile genetic elements,
  <i><a href="http://onlinelibrary.wiley.com/doi/10.1111/biom.12352/abstract" target="_blank">Biometrics</a></i>,
  71, 1009 &#x2013 1021,
	<a href="http://arxiv.org/abs/1411.0031" target="_blank">arXiv:1411.0031</a>, <a href="papers/Xu2015.pdf" target="_blank"><b>PDF</b><a>;
    <a href="papers/Xu_supplement2015.pdf" target="_blank">supplement</a>, software:
   <a href="https://github.com/jasonxu90/bdsem" target="_blank">bdsem</a>.
	 <font color="brown"><b>With a shorter version of this paper, Jason Xu won ASA Biometrics section student travel award to attend JSM 2015.</b></font>

  <li>Lan S, Palacios JA, <b><font color=#8A2BE2>Karcher, M</font></b>, <b>Minin VN</b>, Babak Shahbaba.
  An efficient Bayesian inference framework for coalescent-based nonparametric phylodynamics,
  <i><a href="http://bioinformatics.oxfordjournals.org/content/31/20/3282" target="_blank">Bioinformatics</a></i>, 31, 3282 &#x2013 3289,
  <a href="http://arxiv.org/abs/1412.0158" target="_blank">arXiv:11412.0158</a>,
  <a href="papers/Lan2015.pdf" target="_blank"><b>PDF</b></a>,
  <a href="papers/Lan_supplement2015.pdf" target="_blank">supplement</a>,
  software: <a href="https://github.com/mdkarcher/phylodyn" target="_blank">phylodyn</a>.

  <li><b><font color=#8A2BE2>Xu J</font></b> and <b>Minin VN</b>.
	Efficient transition probability computation for continuous-time branching processes via compressed sensing,
  <i><a href="http://auai.org/uai2015/proceedings/papers/239.pdf" target="_blank">Proceedings of the Thirty-First Conference on Uncertainty in Artificial Intelligence</a></i>,
  952 &#x2013 961.
	(<a href="http://arxiv.org/abs/1503.02644" target="_blank">arXiv:1503.02644</a>),
	  <a href="papers/Xu2015b.pdf" target="_blank"><b>PDF</b></a>,
    <a href="papers/Xu_supplement2015b.pdf" target="_blank">supplement</a>,
	  software: <a href="https://github.com/jasonxu90/bdsem" target="_blank">bdsem</a>.

  <li><b><font color=#8A2BE2>Chi PB</font></b>, Chattopadhyay S, Lemey P, Sokurenko EV, <b>Minin VN</b>.
	Synonymous and nonsynonymous distances help untangle convergent evolution and recombination,
  <i><a href="http://www.degruyter.com/view/j/sagmb.2015.14.issue-4/sagmb-2014-0078/sagmb-2014-0078.xml?format=INT" target="_blank">Statistical Applications in
  Genetics and Molecular Biology</a></i>,
  14, 375 &#x2013 389,
  <a href="http://arxiv.org/abs/1410.1263" target="_blank">arXiv:1410.1263</a>,
	<a href="papers/Chi2015.pdf" target="_blank"><b>PDF</b></a>,
  <a href="papers/Chi_supplement2015.pdf" target="_blank">supplement</a>,
	software: <a href="http://evolmod.r-forge.r-project.org/#synDss" target="_blank">synDss</a>.

  <li>McCoy CO, Bedford T, <b>Minin VN</b>, Robins H, Matsen FA.
	Quantifying evolutionary constraints on B-cell affinity maturation,
	<i><a href="http://m.rstb.royalsocietypublishing.org/content/370/1676/20140244" target="_blank">Philosophical Transactions of the Royal Society B: Biological Sciences</a></i>,
  370, 20140244,
  <a href="http://arxiv.org/abs/1403.3066" target="_blank">arXiv:1403.3066</a>,
	  <a href="papers/McCoy2015.pdf" target="_blank"><b>PDF</b></a>,
	  <a href="papers/McCoy_supplement2015.pdf" target="_blank">supplement</a>,
	  software: <a href="https://github.com/cmccoy/startreerenaissance" target="_blank">startreerenaissance</a>.

  <li><b><font color=#8A2BE2>Lange JM</font></b>, Hubbard RA, Inoue LYT, <b>Minin VN</b>.
  A joint model for multistate disease processes and random informative observation times, with applications to electronic medical records data,
  <i><a href="http://onlinelibrary.wiley.com/doi/10.1111/biom.12252/abstract" target="_blank">Biometrics</a></i>, 71, 90 &#x2013 101,
  <a href="http://biostats.bepress.com/uwbiostat/paper401/" target="_blank">UW Biostatistics Working Paper 401</a>,
  <a href="papers/Lange2015.pdf" target="_blank"><b>PDF</b></a>,
  <a href="papers/Lange_supplement2015.pdf" target="_blank">supplement</a>,
  software: <a href="http://r-forge.r-project.org/R/?group_id=1410" target="_blank">cthmm</a>.

  <li class="year">2014

  <li class="afteryear">Pankey MS, <b>Minin VN</b>, <b><font color=#8A2BE2>Imholte GC</font></b>, Suchard MA, Oakley TH.
  Predictable transcriptome evolution in the convergent and complex bioluminescent organs of squid,
  <i><a href="http://www.pnas.org/content/early/2014/10/21/1416574111" target="_blank">Proceedings of the National Academy of Sciences, USA</a></i>,
  111:E4736 &#x2013 E4742.
  <a href="papers/Pankey2014.pdf" target="_blank"><b>PDF</b></a>,
  <a href="Pankey_supplement2014.pdf" target="_blank">supplement</a>,
  software: <a href="https://github.com/vnminin/indorigin" target="_blank">indorigin</a>.

  <li><b><font color=#8A2BE2>Irvahn J</font></b> and <b>Minin VN</b>.
	Phylogenetic stochastic mapping without matrix exponentiation,
	<i><a href="http://online.liebertpub.com/doi/abs/10.1089/cmb.2014.0062" target="_blank">Journal of Computational Biology</a></i>,
  21:676 &#x2013 690,
	<a href="http://arxiv.org/abs/1403.5040" target="_blank">arXiv:1403.5040</a>,
	<a href="papers/Irvahn2014.pdf" target="_blank"><b>PDF</b></a>,
	<a href="papers/IrvahnSupplement2014.pdf" target="_blank">supplement</a>,
	software: <a href="https://github.com/vnminin/phylomap" target="_blank">phylomap</a>.

  <li><b><font color=#8A2BE2>Chi PB</font></b>, Duncan A, Kramer PA, <b>Minin VN</b>.
	Heritability estimation of osteoarthritis in the pig-tailed macaque (<i>Macaca nemestrina</i>)
	with a look toward future data collection,
	 <i><a href="https://peerj.com/articles/373/" target="_blank">PeerJ</a></i>, 2, e373, <a href="http://dx.doi.org/10.7287/peerj.preprints.284v1"
	 target="_blank">PeerJ PrePrints 2:e284v1</a>, <a href="papers/Chi2014.pdf" target="_blank"><b>PDF</b></a>.
	 <font color="brown"><b>With a shorter version of this paper, Peter Chi was a runner-up in 2012 WNAR student written paper competition.</b></font>

  <li>Leach&eacute AD, Fujita MK, <b>Minin VN</b>, Bouckaert RR.
 	Species delimitation using genome-wide SNP data,
  <i><a href="http://sysbio.oxfordjournals.org/content/63/4/534.abstract" target="_blank">Systematic Biology</a></i>,
  63:534 &#x2013 542,
 	<a href="http://www.biorxiv.org/content/early/2014/01/04/001172" target="_blank">bioRxiv:10.1101/123456</a>,
 	<a href="papers/Leache2014.pdf" target="_blank"><b>PDF</b></a>,
  software: <a href="http://www.beast2.org/wiki/index.php/BFD*" target="_blank">BFD*</a>,
 	<a href="http://sysbio.oxfordjournals.org/content/63/4.cover-expansion" target="blank">issue cover</a>.

  <li>Crawford FW, <b>Minin VN</b>, Suchard MA.
  Estimation for general birth-death processes,
  <i><a href="http://www.tandfonline.com/doi/full/10.1080/01621459.2013.866565" target="_blank">Journal of the American Statistical Association</a></i>,
  109:730 &#x2013 747,
  <a href="http://arxiv.org/abs/1111.4954" target="_blank">arXiv:1111.4954v1</a>,
  <a href="papers/Crawford2014.pdf" target="_blank"><b>PDF</b></a>.

  <li class="year">2013

  <li class="afteryear"><font color=#8A2BE2><b>Doss CR</b></font>, Suchard MA, Holmes I, Kato-Maeda M, <b>Minin VN</b>.
  Fitting birth-death processes to panel data with applications to bacterial DNA fingerprinting,
  <i><a href="http://projecteuclid.org/DPubS?service=UI&version=1.0&verb=Display&handle=euclid.aoas/1387823321" target="_blank">Annals of Applied Statistics</a></i>,
  7, 2315 &#x2013 2335,
  <a href="http://arxiv.org/abs/1009.0893" target="_blank">arXiv:1009.0893</a>,
  <a href="papers/doss_et_al2013.pdf" target="_blank"><b>PDF</b></a>,
  <a href="papers/doss_et_al2013_supplement.pdf" target="_blank">supplement</a>,
  software: <a href="http://cran.r-project.org/web/packages/DOBAD/index.html" target="_blank">DOBAD</a>.

  <li>Leach&eacute AD, Palacios JA, <b>Minin VN</b>, Bryson RW.
  Phylogeography of the Trans-Volcanic bunchgrass lizard (<i>Sceloporus bicanthalis</i>) across the highlands of southeastern Mexico,
  <i><a href="http://onlinelibrary.wiley.com/doi/10.1111/bij.12172/abstract" target="_blank">Biological Journal of the Linnean Society</a></i>,
  110, 852 &#x2013 865,
  <a href="papers/leache_et_al2013.pdf" target="_blank"><b>PDF</b></a>.

  <li><font color=#8A2BE2><b>Lange JM</b></font> and <b>Minin VN</b>.
  Fitting and interpreting continuous-time latent Markov models for panel data,
  <i><a href="http://onlinelibrary.wiley.com/doi/10.1002/sim.5861/abstract" target="_blank">Statistics in Medicine</a></i>,
  32, 4581 &#x2013 4595,
  <a href="http://biostats.bepress.com/uwbiostat/paper382/" target="_blank">UW Biostatistics Working Paper 382</a>,
  <a href="papers/lange_minin2013.pdf" target="_blank"><b>PDF</b><a>,
  <a href="papers/lange_minin2013_supplement.pdf" target="_blank">supplement</a>,
  software: <a href="http://r-forge.r-project.org/R/?group_id=1410" target="_blank">cthmm</a>.
  <font color="brown"><b>With a shorter version of this paper, Jane Lange won 2012 WNAR student written paper competition.</b></font>

  <li><font color=#8A2BE2><b>Irvahn J</b></font>, Chattopadhyay S, Sokurenko EV, <b>Minin VN</b>.
  rbrothers: R package for Bayesian multiple change-point recombination detection,
  <i><a href="http://www.la-press.com/rbrothers-r-package-for-bayesian-multiple-change-point-recombination-d-article-a3718" target="_blank">Evolutionary Bioinformatics</a></i>,
  9:235 &#x2013 238,
  <a href="papers/jirvahn_et_al2013.pdf" target="_blank"><b>PDF</b></a>,
  software: <a href="http://evolmod.r-forge.r-project.org/" target="_blank">rbrothers</a>.

  <li><font color=#8A2BE2><b>Palacios JA</b></font> and <b>Minin VN</b>.
  Gaussian process-based Bayesian nonparametric inference of population trajectories from gene genealogies,
  <i><a href="http://onlinelibrary.wiley.com/doi/10.1111/biom.12003/abstract" target="_blank">Biometrics</a></i>,
  69, 8 &#x2013 18,
  <a href="http://arxiv.org/abs/1112.4138" target="_blank">arXiv:1112.4138</a>,
  <a href="papers/palacios_minin2013.pdf" target="_blank"><b>PDF</b></a>,
  <a href="papers/palacios_minin2013_supplement.pdf" target="_blank">supplement</a>.

  <li class="year">2012

  <li class="afteryear">Lemey P, <b>Minin VN</b>, Bielejec F, Kosakovsky Pond SL, Suchard MA.
  A counting renaissance:
  Combining stochastic mapping and empirical Bayes to quickly detect amino acid sites under positive
  selection,
  <i><a href="http://bioinformatics.oxfordjournals.org/content/28/24/3248" target="_blank">Bioinformatics</a></i>,
  28, 3248 &#x2013 3256,
  <a href="papers/Lemey2012.pdf" target="_blank"><b>PDF</b></a>,
  software: <a href="http://beast.bio.ed.ac.uk" target="_blank">BEAST</a>.

  <li> <font color=#8A2BE2><b>Ryu S</b></font>, Goodlett DR, Noble WS, <b>Minin VN</b>.
  A statistical approach to peptide identification from clustered tandem mass spectrometry data,
  <i><a href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?tp=&arnumber=6470214&contentType=Conference+Publications&searchField%3DSearch_All%26queryText%3DSOYOUNG+RYU" target="_blank">2012 IEEE International Conference on Bioinformatics and Biomedicine Workshops (BIBMW)</a></i>,
  648 &#x2013 653,
  <a href="papers/Ryu2012.pdf" target="_blank"><b>PDF</b></a>.

  <li> <font color=#8A2BE2><b>Palacios JA</b></font> and <b>Minin VN</b>.
	Integrated nested Laplace approximation for Bayesian nonparametric phylodynamics,
	<i><a href="http://www.auai.org/uai2012/papers/310.pdf" target="_blank">Proceedings of the Twenty-Eighth Conference on Uncertainty in Artificial Intelligence</a></i>,
  726 &#x2013 735,
	<a href="papers/palacios_minin2012.pdf" target="_blank"><b>PDF</b></a>.

  <li><font color=#8A2BE2><b>Sawaya SM</b></font>*, <font color=#8A2BE2><b>Lennon D</b></font>*, Buschiazza G, Gemmell N, <b>Minin VN</b>.
    Measuring microsatellite conservation in mammalian evolution with a phylogenetic birth-death model,
  <i><a href="http://gbe.oxfordjournals.org/content/4/6/636" target="_blank">Genome Biology and Evolution</a></i>,
  4, 636 &#x2013 647,
  <a href="papers/Sawaya2012.pdf" target="_blank"><b>PDF</b></a>,
  <a href="sawaya2012_supp/index.html" target="_blank">supplement</a>,
  software: <a href="https://github.com/dnlennonpu01/McSMAC" target="_blank">McSMAC</a>.
  * - joint first authors

  <li> Kemal K, Kitchen C, Burger H, Foley B, Klimkait T, Hamy F, Anastos, K, Petrovic K, <b>Minin VN</b>,
  Suchard MA, Weiser B.
  Recombination between variants from genital tract and plasma: Evolution of multidrug-resistant HIV-1,
  <i><a href="http://online.liebertpub.com/doi/abs/10.1089/AID.2011.0383" target="_blank">AIDS Research and Human Retroviruses</a></i>,
  28, 1766 &#x2013 1774,
  <a href="papers/Kemal2012.pdf" target="_blank"><b>PDF</b></a>.

  <li class="year">2011

  <li class="afteryear"><b>Minin VN</b>, O'Brien JD, <font color=#8A2BE2><b>Seregin A</b></font>.
  Imputation estimators partially correct for model misspecification,
  <i><a href="http://www.degruyter.com/view/j/sagmb.2011.10.issue-1/sagmb.2011.10.1.1650/sagmb.2011.10.1.1650.xml?format=INT" target="_blank">Statistical Applications in Genetics and Molecular Biology</a></i>,
  10, 1 &#x2013 24,
  an earlier version with a beta-binomial example: <a href="http://arxiv.org/abs/0911.0930v1" target="_blank">arXiv:0911.0939v1</a>,
  <a href="papers/Minin2011.pdf" target="_blank"><b>PDF</b></a>.

  <li class="year">2009

  <li class="afteryear">Chattopadhyay S, Weissman SJ, <b>Minin VN</b>, Russo TA, Dykhuizen DE, Sokurenko EV.
  High frequency of hotspot mutations in core genes of Escherichia coli due to short-term positive
  selection,
  <i><a href="http://www.pnas.org/content/106/30/12412.abstract" target="_blank">Proceedings of the National Academy of Sciences, USA</a></i>,
  106, 12412 &#x2013 12417, <a href="papers/Chattopadhyay2009.pdf" target="_blank"><b>PDF</b></a>.

  <li> O'Brien JD*, <B>Minin VN</B>*, and Suchard MA.
    Learning to count: robust estimates for labeled distances between
    molecular sequences,
  <i><a href="http://mbe.oxfordjournals.org/cgi/content/abstract/26/4/801" target="blank">Molecular Biology and Evolution</a></i>,
  26, 801 &#x2013 814, <a href="papers/OBrien2009.pdf" target="_blank"><b>PDF</b></a>,
  software: <a href="http://www.stat.washington.edu/vminin/markovjumps/" target="_blank">markovjumps</a>.
  * - joint first authors

  <li class="year">2008

  <li class="afteryear"><b>Minin VN</b> and Suchard MA.
  Fast, accurate and simulation-free stochastic mapping,
  <i><a href="http://rstb.royalsocietypublishing.org/content/363/1512/3985.abstract" target="_blank">Philosophical Transactions of the Royal Society B: Biological Sciences</a></i>,
  363, 3985 &#x2013 3995, <a href="papers/MininSuchard2009.pdf" target="_blank"><b>PDF</b></a>, <a href="papers/MininSuchard2009_full.pdf" target="_blank">an earlier version with more details</a>.

  <li><b>Minin VN</b>, Bloomquist EW, and Suchard MA.
  Smooth skyride through a rough skyline: Bayesian coalescent-based inference of population dynamics,
  <i><a href="http://mbe.oxfordjournals.org/cgi/content/abstract/msn090?ijkey=yPxR6ZltFJv8hUz&keytype=ref"
  target="_blank">Molecular Biology and Evolution</a></i>, 25, 1459 &#x2013 1471,
  <a href="papers/Minin2008b.pdf" target="_blank"><b>PDF</b></a>, software: <a href="http://beast.bio.ed.ac.uk/Main_Page" target="_blank">BEAST</a>.

  <li><b>Minin VN</b> and Suchard MA.
  Counting labeled transitions in continuous-time Markov models of evolution,
  <i><a href="http://www.springerlink.com/content/9140j00427823331/" target="_blank">Journal of Mathematical Biology</a></i>,
  56, 391 &#x2013 412, <a href="papers/Minin2008a.pdf" target="_blank"><b>PDF</b></a>.

  <li class="year">2007

  <li class="afteryear">Rajaram ML, <b>Minin VN</b>, Suchard MA, and Dorman KS.
  Hot and cold: spatial fluctuation in HIV-1 recombination rates,
  <i><a href="http://ieeexplore.ieee.org/xpl/freeabs_all.jsp?arnumber=4375638" target="_blank">Proceedings of the 7th IEEE International Conference on Bioinformatics and
  Bioengineering</a></i>,
  707 &#x2013 714, <a href="papers/Rajaram2007.pdf" target="_blank"><b>PDF</b></a>.

  <li><b>Minin VN</b>, Dorman KS, Fang F, and Suchard MA.
  Phylogenetic mapping of recombination hot-spots in HIV via spatially smoothed change-point processes,
  <i><a href = "http://www.genetics.org/cgi/content/abstract/genetics.106.066258v1" target="_blank">Genetics</a></i>,
  175, 1773 &#x2013 1785, <a href="papers/Minin2007.pdf" target="_blank"><b>PDF</b></a>,
  <a href="http://www.genetics.org/cgi/content/full/175/4/NP?maxtoshow=&hits=10&RESULTFORMAT=&fulltext=issue+highlights+April+2007&andorexactfulltext=and&searchid=1&FIRSTINDEX=0&sortspec=relevance&resourcetype=HWCIT" target="_blank">highlighted article</a>.

  <li>Fang F, Ding J, <b>Minin VN</b>, Suchard MA, and Dorman KS.
  cBrother: Relaxing parental tree assumptions for Bayesian recombination detection,
  <i><a href = "http://bioinformatics.oxfordjournals.org/cgi/content/abstract/23/4/507" target="_blank">Bioinformatics</a></i>,
  23, 507 &#x2013 508, <a href="papers/Fang2007.pdf" target="_blank"><b>PDF</b></a>,
  software: <a href="http://thirteen-01.stat.iastate.edu/software/index.xml#cbrother" target="_blank">cBrother</a>.

  <li class="year">2005

  <li class="afteryear"><b>Minin VN</b>, Dorman KS, Fang F, and Suchard MA.
  Dual multiple change-point model leads to more accurate recombination detection,
  <i><a href = "http://bioinformatics.oxfordjournals.org/cgi/content/abstract/21/13/3034" target="_blank">Bioinformatics</a></i>,
  21,3034 &#x2013 3042, <a href="papers/Minin2005.pdf" target="_blank"><b>PDF</b></a>,
  software: <a href="http://www.biomath.ucla.edu/msuchard/DualBrothers/" target="_blank">DualBrothers<a>.

  <li>Abdo Z, <b>Minin VN</b>, Joyce P, and Sullivan J.
  Accounting for uncertainty in the tree topology has little effect on the decision theoretic approach to model selection in phylogeny estimation,
  <i><a href = "http://mbe.oxfordjournals.org/cgi/content/abstract/22/3/691" target="_blank">Molecular Biology and Evolution</a></i>,
  22, 691 &#x2013 703, <a href="papers/Abdo2005.pdf" target="_blank"><b>PDF</b></a>.

  <li class="year">2004

  <li class="afteryear">Liu X, <b>Minin V</b>, Huang Y, Selingson D, and Horvath S.
  Statistical methods for analyzing tissue microarray data,
  <i><a href = "http://www.informaworld.com/smpp/content~content=a713617062~db=all~order=page" target="_blank">Journal of Biopharmaceutical Statistics</a></i>,
  14, 671 &#x2013 685, <a href="papers/TissueMicroarray.pdf" target="_blank"><b>PDF</b></a>.

  <li class="year">2003

  <li class="afteryear"><b>Minin V</b>, Abdo Z, Joyce P, and Sullivan J.
  Performance-based selection of likelihood models for phylogeny estimation,
  <i><a href = "http://sysbio.oxfordjournals.org/content/52/5/674.short" target="_blank">Systematic Biology</a></i>,
  52, 674 &#x2013 683, <a href="papers/Minin2003.pdf" target="_blank"><b>PDF</b></a>,
  software: <a href="http://www.webpages.uidaho.edu/~jacks/DTModSel.html" target="_blank">DTModSel</a>.

</ol>


Last modified: August, 2017


</div>

<div class="col-2">

</div>
</div>
</body> </html>
